priam/src/main/java/com/netflix/priam/backup/BackupRestoreUtil.java

```diff
@@ -133,22 +129,12 @@ public final boolean isFiltered(String keyspace, String columnFamilyDir) {
             if (excludeFilter.containsKey(keyspace)
                     && (excludeFilter.get(keyspace).isEmpty()
                             || excludeFilter.get(keyspace).contains(columnFamilyName))) {
-                logger.debug(
```

Comment:
I suppose it's fine and you have a better understanding around whether or not we ever need those log messages but in general I'm not opposed to debug logs that I can enable to help debug an issue